services/government-frontend/docker-compose.yml

```diff
@@ -44,3 +44,17 @@ services:
       VIRTUAL_HOST: draft-government-frontend.dev.gov.uk
       PLEK_HOSTNAME_PREFIX: "draft-"
       HOST: 0.0.0.0
+
+  government-frontend-real:
+    <<: *government-frontend
+    depends_on:
+      - nginx-proxy
+    environment:
+      GOVUK_WEBSITE_ROOT: https://www.gov.uk
+      PLEK_SERVICE_CONTENT_STORE_URI: https://www.gov.uk/api
+      PLEK_SERVICE_STATIC_URI: assets.publishing.service.gov.uk
+      VIRTUAL_HOST: real-government-frontend.dev.gov.uk
```

Comment:
maybe `www-government-frontend.dev.gov.uk` is better than real.

Reply:
Happy for any other suggestions. `prod-government-frontend.dev.gov.uk` or even `live-government-frontend.dev.gov.uk`.

Reply:
We could just go with 'government-frontend', which just leaves the stack name to decide on. Does that help?

Reply:
👍 for keeping the domain `government-frontend.dev.gov.uk` for all different stacks, you're not likely to run 2 different versions at the same time.